go/cmd/vtctldclient/command/query.go

```diff
@@ -62,6 +62,13 @@ var (
 		Args:  cobra.ExactArgs(1),
 		RunE:  commandGetUnresolvedTransactions,
 	}
+	// ConcludeTransaction makes a ConcludeTransaction gRPC call to a vtctld.
+	ConcludeTransaction = &cobra.Command{
+		Use:   "ConcludeTransaction <dtid> [<keyspace/shard> ...]",
+		Short: "Concludes the unresolved transaction by rolling back any prepared transaction and deleting the transaction metadata record.",
```

Comment:
```suggestion
		Short: "Concludes the unresolved transaction by rolling back any prepared transactions and deleting the transaction metadata record.",
```

Reply:
This works on a single distributed transaction. I can change some wording here.